releng/org.eclipse.n4js.targetplatform/org.eclipse.n4js.targetplatform.target

```diff
@@ -56,15 +67,10 @@
       <unit id="com.fasterxml.jackson.core.jackson-databind" version="2.9.2.v20180207-1730"/>
       <unit id="com.google.guava" version="21.0.0.v20170206-1425"/>
       <unit id="com.google.inject" version="3.0.0.v201605172100"/>
+      <unit id="javax.xml.bind" version="2.2.0.v201105210648"/>
```

Comment:
Is there a more recent version available somewhere?

Reply:
javax.xml.bind 2.2.0 is the the latest version in the Orbit https://download.eclipse.org/tools/orbit/downloads/drops/R20190226160451/